MApp.js

```diff
@@ -0,0 +1,110 @@
+document.addEventListener('DOMContentLoaded', () => {
+   fetch('https://ghibliapi.herokuapp.com/films') //we want the data here before user interacton
```

Comment:
Another neat way of doing this would be putting fetch function inside a function. Not necessary in small apps like this but definitely useful way of organizing your code in the future.